roles/kubernetes/master/defaults/main.yml

```diff
@@ -26,6 +26,7 @@ force_etcd3: false
 
 # audit support
 kubernetes_audit: false
+# audit_log_path must not be set to "-" with kubeadm as it only handles a logfile named audit.log
```

Comment:
could you open an issue in kubeadm or link if it exist already ? 
So we know when this no more the case

Reply:
@ant31 it seems that AuditPolicy won't stay in kubeadm root config (https://github.com/kubernetes/kubeadm/issues/1029#issuecomment-409742752).
Maybe the best option to setup auditing in kubeadm is finally to explicitly add extravolumes and define extraapiserverflags what do you think?

Reply:
BTW there was a question on slack https://kubernetes.slack.com/archives/C2V9WJSJD/p1534530725000100 about having a custom policy. I guess I could add a audit_policy_data variable and turn the policy file into a j2 template. What do you think?